js/table.js

```diff
@@ -33,7 +33,7 @@ class Table {
       var elem = document.getElementById('notification-block')
       elem.parentNode.removeChild(elem)
     }
-
+    this.cards.length = 0 ? this.buildCards() : this.cards.length = 0
```

Comment:
Kas šeit notiek?
Ja pārbaudām, tad vajag ar vismaz 2 ==, šeit sanāk ,ka mēs uzreiz piešķiram vērtību. Kāda bija ideja šai funkcijai?

Reply:
Jā, tur uzreiz notiek masīva dzēšana, tiešām kļūdījos - ternārā operatora pirmajā daļā ir jābūt pārbaudei - vai masīvs ir tukšs, 
this.cards.length == 0 ? this.buildCards() : this.cards.length = 0

Reply:
@arvids-dane Kāda ir ideja/vajadzība dotajai funkcijai? Ja mums nav kāršu, tad tās ir jāizveido?

Reply:
Plus tas this.cards.length interesants, strādā cik es skatos.
Bet varētu varbūt:
```js
this.cards = []
```
Tad uzreiz skaidrāk, manuprāt. Bet šādi arī var palikt.

Reply:
Šai funkcijai ir ideja - lai novērstu attēlāredzmo situāciju, kad ar katru nākamo raundu kāršu kavu skaits palielinās, saite uz bildi https://drive.google.com/file/d/1elrW8u1UkESNJ2IE-xJQ-Yuuxo2OiT_m/view?usp=sharing

Reply:
@arvids-dane Cik es saprotu ,tad mēs gribam izveidot kārtis, ja to vairs nav?
Tad var šādi:
```js
this.cards.length === 0 && this.buildCards()
```
Ko tas dara? Ja pirmais nosacījums izpildās un mums ir tieši 0 kāršu, tad mēs taisām no jauna.